charging_stations_pipelines/pipelines/osm/osm_receiver.py

```diff
@@ -45,4 +70,4 @@ def get_osm_data(country_code: str, tmp_data_path):
 
 
 if __name__ == "__main__":
-    get_osm_data(tmp_data_path="./osm_france.json")
+    get_osm_data("FR", "./osm_france.json")
```

Comment:
Is this main function needed? I think we can remove it.

Reply:
Typically used for local testing. Should be removed.

Reply:
OK, I removed call as main.